api/src/job.js

```diff
@@ -218,15 +247,18 @@ class Job {
                 null;
             this.#active_timeouts.push(kill_timeout);
 
+            // when the process writes to stderr, send the data to the event_bus (over websocket later)
             proc.stderr.on('data', async data => {
                 if (event_bus !== null) {
                     event_bus.emit('stderr', data);
-                } else if ((stderr.length + data.length) > this.runtime.output_max_size) {
+                } else if (
+                    stderr.length + data.length >
```

Comment:
A little personal thing for me is putting expressions before comparing operators like this in brackets - in my mind it helps clear up order of operations.